velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -50,6 +50,24 @@ std::unique_ptr<::duckdb::ConstantFilter> constantEqualFilter(
       ::duckdb::ExpressionType::COMPARE_EQUAL, std::move(value));
 }
 
+void buildConjunctOrFilter(
```

Comment:
It would be nice to extract this change into a separate PR and add a test to velox/dwio/parquet/tests/ParquetReaderTest.cpp

Reply:
https://github.com/facebookincubator/velox/pull/1472